titus-common/src/main/java/com/netflix/titus/common/util/feature/FeatureGuards.java

```diff
@@ -19,10 +19,31 @@
 import java.util.function.Function;
 import java.util.function.Predicate;
 
+import com.netflix.titus.common.util.feature.FeatureGuard.FeatureGuardResult;
+
 public class FeatureGuards {
 
-    public static <T> Predicate<T> toPredicate(FeatureGuard<T> featureGuard) {
-        return value -> featureGuard.matches(value) == FeatureGuard.FeatureGuardResult.Approved;
+    public static <T> Predicate<T> toPredicate(FeatureGuard<T>... featureGuard) {
```

Comment:
Can you document the intended behavior of this method? I can see it being confusing due to having to figure out how multiple feature guards are composed together.